src/Files/App.xaml.cs

```diff
@@ -259,6 +259,8 @@ protected override async void OnLaunched(LaunchActivatedEventArgs e)
                     }
                 }
             }
+
+            WindowDecorationsHelper.RequestWindowDecorationsAccess();
```

Comment:
To be put also in OnActivated and OnFileActivated?

Reply:
I think it only needs to happen once in the app lifecycle.

Reply:
Oh, so is OnActivated always called?
If so we can move that call to be only there

Reply:
No I think we must put it in all 3 places. The method called depends how the app is launched.